packages/lesswrong/components/search/UsersSearchAutocompleteHit.tsx

```diff
@@ -2,38 +2,36 @@ import {Components, registerComponent} from '../../lib/vulcan-lib'
 import React from 'react'
 import {styles as metaInfo} from '../common/MetaInfo'
 
-const specificityFmClass = 'forum-magnum'
-const specificityCkClass = 'ck-override'
 const styles = (theme: ThemeType): JssStyles => ({
   userHitLabel: {
-    // A specificity hack to work around https://github.com/ckeditor/ckeditor5/issues/3424
-    [`&.${specificityFmClass}.${specificityCkClass}, &.${specificityFmClass}.${specificityCkClass} *`]: {
-      ...theme.typography.body2,
-      ...metaInfo(theme).root,
-      marginLeft: theme.spacing.unit,
+    ...theme.typography.body2,
+    ...metaInfo(theme).root,
+    marginLeft: theme.spacing.unit,
+
+    // To properly switch color on item being selected
+    [`.ck-on &`]: {
+      color: 'inherit',
```

Comment:
Is the css specificity hack that was previously here no longer needed?

Reply:
I found a better way of doing this, by adding `ck-reset_all-excluded` class to a parent